### PR TITLE
EBS autoscaling deprecation

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Update AWS Batch CE docs to deprecate EBS autoscaling, per https://github.com/seqeralabs/platform/pull/7088

### CODE REVIEW

**Comment by llewellyn-sl** [comment]
@mbosio85 can you let us know when the EBS deprecation will be live? This PR will need to merge in time for that.

**Comment by gwright99** [comment]
@nate-simon and I will be speaking to a client tomorrow and this is likely to come up - would be helpful to have a timeline.

**Comment by mbosio85** [comment]
@llewellyn-sl @gwright99 from the [PR information](https://github.com/seqeralabs/platform/pull/7088) the deprecation in the UI (no functional change now) is part of the [v24.1.0 milestone](https://github.com/seqeralabs/platform/milestone/95?closed=1).
I hope this clarify when the UI hint is in place. Then, for functional impact, that will be at least 6 months away as mentioned in https://github.com/seqeralabs/tower-backlog/issues/1283

**Comment by gwright99** [comment]
> @llewellyn-sl @gwright99 from the [PR information](https://github.com/seqeralabs/platform/pull/7088) the deprecation in the UI (no functional change now) is part of the [v24.1.0 milestone](https://github.com/seqeralabs/platform/milestone/95?closed=1). I hope this clarify when the UI hint is in place. Then, for functional impact, that will be at least 6 months away as mentioned in [seqeralabs/tower-backlog#1283](https://github.com/seqeralabs/tower-backlog/issues/1283)

Thanks - we encouraged them to switch over to Fusion v2 (_to let the Fusion garbage collection self-cleanup use the NVME disk efficiently_). 

They also still use the script I gave them a few years ago which allows after-the-fact modifications to AWS Batch instances, so they've got a way to stick with EBS auto-scaling if it turns out all the new solutions don't work for their situation.

**Comment by llewellyn-sl** [comment]
@mbosio85 should we merge this for 24.1?

File: platform_versioned_docs/version-23.4.0/compute-envs/aws-batch.mdx [hunk 1]
```diff
@@ -139,7 +139,7 @@ You can create a compute environment that launches either spot or on-demand inst
 :::
 
 15. Enter the **Max CPUs**, e.g., `64`. This is the maximum number of combined CPUs (the sum of all instances' CPUs) AWS Batch will provision at any time.
-16. Select **EBS Auto scale** to allow the EC2 virtual machines to dynamically expand the amount of available disk space during task execution.
+16. Select **EBS Auto scale (deprecated)** to allow the EC2 virtual machines to dynamically expand the amount of available disk space during task execution. This feature deprecated, and is not compatible with Fusion v2.
```
Comment (mbosio85) [inline platform_versioned_docs/version-23.4.0/compute-envs/aws-batch.mdx:142]: It is missing an "is" before deprecated

Comment (llewellyn-sl) [inline platform_versioned_docs/version-23.4.0/compute-envs/aws-batch.mdx:142]:
```suggestion
16. Select **EBS Auto scale (deprecated)** to allow the EC2 virtual machines to dynamically expand the amount of available disk space during task execution. This feature is deprecated, and is not compatible with Fusion v2.
```

**Comment by llewellyn-sl** [comment]
Ported changes to 24.1 enterprise branch. Closing.

**Pull request closed without merging**